cmd/sharded-test-server/frontproxy.go

```diff
@@ -218,15 +218,15 @@ func kcpAdminKubeConfig(ctx context.Context, hostIP string) error {
 			Server:               baseHost + "/clusters/root:default",
 			CertificateAuthority: ".kcp/serving-ca.crt",
 		},
-		"system:admin": {
+		"base": {
 			Server:               baseHost,
 			CertificateAuthority: ".kcp/serving-ca.crt",
 		},
 	}
 	kubeConfig.Contexts = map[string]*clientcmdapi.Context{
-		"root":         {Cluster: "root", AuthInfo: "admin"},
-		"default":      {Cluster: "root:default", AuthInfo: "admin"},
-		"system:admin": {Cluster: "system:admin", AuthInfo: "admin"},
+		"root":    {Cluster: "root", AuthInfo: "kcp-admin"},
+		"default": {Cluster: "root:default", AuthInfo: "kcp-admin"},
```

Comment:
you have to rebase? this context is gone in main.

Reply:
oh, maybe it was just forgotten. Can you remove it? The default org is no more.

Reply:
can you point me to what you mean with "gone" 🤔